http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/AzureDependenciesBuilder.scala

```diff
@@ -0,0 +1,94 @@
+package org.broadinstitute.dsde.workbench.leonardo.http
+import akka.actor.ActorSystem
+import cats.effect.{IO, Resource}
+import org.broadinstitute.dsde.workbench.leonardo.config.Config.{appServiceConfig, gkeCustomAppConfig}
+import org.broadinstitute.dsde.workbench.leonardo.db.DbReference
+import org.broadinstitute.dsde.workbench.leonardo.http.service.LeoAppServiceInterp
+import org.broadinstitute.dsde.workbench.leonardo.monitor.MonitorAtBoot
+import org.broadinstitute.dsde.workbench.leonardo.util.ServicesRegistry
+import org.broadinstitute.dsde.workbench.openTelemetry.OpenTelemetryMetrics
+import org.typelevel.log4cats.StructuredLogger
+
+import scala.concurrent.ExecutionContext
+
+class AzureDependenciesBuilder extends CloudDependenciesBuilder {
+
+  /**
+   * Registers the OT tracing for the cloud hosting provider.
+   *
+   * @return
+   */
+  // TODO: we must implement this for Azure.
```

Comment:
is this TODO relevant? Should we have a ticket?

Reply:
Yes, it is relevant. It requires design work of where to send the metrics when hosting on Azure.  This is follow up we are planning to take as well.  The method should never be called when Leo is hosted in GCP.